src/style.css

```diff
@@ -1 +1,110 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 400;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+  font-style: normal;
+}
+
+body {
+  margin-top: 20px;
+}
+
+/* #region form */
+
+.form {
+  display: flex;
+  position: relative;
+  flex-direction: column;
+  gap: 20px;
+}
+
+.form__bar {
+  /* element settings */
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-family: Avenir, Helvetica, sans-serif;
+
+  /* content ssetings */
+  color: #3d4e61;
+  font-weight: 400;
+}
+
+.form__bar--big {
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+}
+
+.form__bar--small {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+}
+
+.form__bar--big::placeholder {
+  font-weight: 400;
+  color: #3d4e61;
+}
+
+.form__bar--small::placeholder {
+  font-weight: 400;
+  color: #3d4e61;
+}
+
+.form__bar--big:hover {
+  box-shadow: 0 3px 8px 0 #3d4e6133;
+}
+
+.form__bar--small:hover {
+  box-shadow: 0 3px 8px 0 #3d4e6133;
+}
+
+.form__bar--big:focus {
+  outline: none;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  box-shadow: (0 4px 4px #3d4e6133);
```

Comment:
There seems to be a typo in the box-shadow value. The correct syntax for box-shadow does not include parentheses around the values. It should be `box-shadow: 0 4px 4px rgba(61, 78, 97, 0.2);`.